src/components/modules/header/Header.tsx

```diff
@@ -51,15 +53,45 @@ const subMessageStyles = css`
 
 export const Header: React.FC<HeaderProps> = (props: HeaderProps) => {
     const { heading, subheading, primaryCta, secondaryCta } = props.content;
+    const { abTestName, abTestVariant, componentType, campaignCode } = props.tracking;
+
+    const sendOphanEvent = (action: OphanAction): void =>
+        props.submitComponentEvent &&
+        props.submitComponentEvent({
+            component: {
+                componentType,
+                id: campaignCode,
+                campaignCode,
+            },
+            action,
+            abTest: {
+                name: abTestName,
+                variant: abTestVariant,
+            },
+        });
+
+    const [hasBeenSeen, setNode] = useHasBeenSeen(
+        {
+            threshold: 0,
+        },
+        true,
+    ) as HasBeenSeen;
+
+    useEffect(() => {
+        if (hasBeenSeen) {
+            sendOphanEvent('VIEW');
+        }
+    }, [hasBeenSeen]);
```

Comment:
If wonder if we use this pattern enough, maybe in a future PR we could factor out a `useHasBeenSeenEffect` that takes a callback similar to `useEffect` but just runs it once when the component has been seen. e.g 

```ts
useHasBeenSeenEffect(() => sendOphanEvent('VIEW'));
```

Just musing really!

Reply:
Good point. The `useHasBeenSeen` hook already gets us part way there, it can probably be replaced with your suggestion. I'll do this in a follow up